parser_library/src/processing/opencode_provider.cpp

```diff
@@ -202,7 +202,15 @@ void opencode_provider::feed_line(const parsing::parser_holder& p, bool is_proce
 {
     m_line_fed = true;
 
-    p.input->reset(m_current_logical_line);
+    const auto& subs = p.input->new_input(m_current_logical_line);
+
+    if (subs.server)
+        m_diagnoser->add_diagnostic(
+            diagnostic_op::warning_W017(range(position(m_current_logical_line_source.begin_line, 0))));
+
+    if (subs.client)
+        m_diagnoser->add_diagnostic(
+            diagnostic_op::warning_W018(range(position(m_current_logical_line_source.begin_line, 0))));
```

Comment:
I wonder if this should be limited to one message per opencode or something similar... just image the number of messages if you open something semi-binary by accident...

Reply:
Good point!